tests/integration/routes/accounts/get-accounts-test.js

```diff
@@ -54,8 +55,20 @@ getServer(function (error, server) {
       t.end()
     })
 
-    group.test('Not an admin', {todo: true}, function (t) {
-      t.end()
+    group.test('Not an admin', function (t) {
+      var requestOptions = _.defaultsDeep({
+        headers: {
+          authorization: 'Bearer cGF0LWRvZToxMjc1MDA6zEZsQ1BuO-W8SthDSrg8KXQ8OlQ'
```

Comment:
Yes, excellent point. `cGF0LWRvZToxMjc1MDA6zEZsQ1BuO-W8SthDSrg8KXQ8OlQ` is the Session ID as CouchDB calculates it from username ('pat-doe'), salt ('salt123'), server secret ('secret') and a unix timestamp (1209600). See 
https://tonicdev.com/56fbf3cf812b1112005f701c/56fbf3d496866f11009cb78e

We could add a comment like

```js
// Session ID based on 'pat-doe', 'salt123', 'secret', 1209600
```

Would that make sense?